feature-libs/asm/customer-360/components/customer-360/avatar-label.pipe.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import { User } from '@spartacus/core';
+
+@Pipe({ name: 'avatarLabel' })
```

Comment:
Nitpick, but almost all of pipes names in the storefront starts from "cx".  Maybe for consistency we should add "cx" prefix also.